initfiles/componentfiles/thor/run_thor

```diff
@@ -43,6 +42,13 @@ while [ 1 ]; do
     fi
     $deploydir/makethorgroup
     sort $instancedir/slaves | uniq > $instancedir/uslaves.start
+    diff <(daliadmin server=$DALISERVER dfsgroup ${groupName} | sort | uniq) <(${INSTALL_DIR}/sbin/configgen -env ${CONFIG_DIR}/${ENV_XML_FILE} -c ${groupName} -listall2 | awk 'BEGIN {FS=",";} /ThorSlaveProcess/ { print $3 }' | sort | uniq) &>/dev/null
```

Comment:
configgen should use -c ${THORNAME) not ${groupName}
The group name is not necessarily the same as the thor name.